src/components/input/select/index.tsx

```diff
@@ -141,14 +141,22 @@ export class SmoothlyInputSelect implements Input, Editable, Clearable, Componen
 	@Listen("click")
 	onClick(event: UIEvent): void {
 		event.stopPropagation()
```

Comment:
I know this is not part of the PR but generally stopping to propagation of click events might make this component very annoying to use. try to avoid it.